playbooks/deploy_user.yml

```diff
@@ -1,4 +1,7 @@
 ---
+# This playbook should not be run on all hosts during business hours.  
+# If you are running it outside the maintenance window please run with a limit flag.
```

Comment:
Thank you for adding this warning! Can we say why it shouldn't be run during business hours?

Reply:
Yes, this means that no one can deploy while it is running.  Out thought was to not run it during business hours because of this.  That can be added to the comment.